### ISSUE TITLE
REST API: json character encoding

### ISSUE THREAD

**Issue by pietervanleeuwen** [linkedtv/platform#13]
When requesting annotations through the API using json the text characters encoding is not always correct.
For example annotation ebfad895-2096-4d38-b80b-9729b9ad858f US-Präsidenten contains an ä.  When requested through json the response is encoded incorrectly encoded as 'US-PrÃ¤sidenten'

**Comment by alisarioglu** [comment]
Hi Pieter, I think you see this characters because your client (windows command line, ...), is not be able to show utf-8. Try the following curl request and open the file with an editor that can handle utf-8.

curl -X GET http://data.linkedtv.eu/API/annotation/ebfad895-2096-4d38-b80b-9729b9ad858f?_view=full -v --header "Accept:application/json;" > utf-8-file.txt

**Comment by jthomsen** [comment]
hi pieter, please check, cheers jan

**Comment by pietervanleeuwen** [comment]
I've checked and Ali was right that the encoding was UTF-8. 
The error came from the fact that we rely on what the servers response header content-type is telling what the encoding is (using http://hc.apache.org/httpclient-legacy/apidocs/org/apache/commons/httpclient/HttpMethodBase.html#getResponseCharSet%28%29).
However the API doesn't provide any content-type in the response header at all and for that reason our app defaulted to ISO-8859-1. We fixed it internally, but it would also be nice if the API tells the encoding in the content-type header.
